seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/AbstractReadStrategy.java

```diff
@@ -153,15 +151,10 @@ public List<String> getFileNamesByPath(HadoopConf hadoopConf, String path) throw
                 }
             }
         }
-
-        if (fileNames.isEmpty()) {
-            throw new FileConnectorException(
-                    FileConnectorErrorCode.FILE_LIST_EMPTY,
-                    "The target file list is empty,"
-                            + "SeaTunnel will not be able to sync empty table, "
-                            + "please check the configuration parameters such as: [file_filter_pattern]");
+        if (this.fileNames.isEmpty()) {
+            log.error("The current directory is empty " + path);
+            this.fileNames.add(path);
```

Comment:
This place looks strange. If there are no files in the directory, is it more appropriate to return an empty file list? I think the reader of the file list should handle the empty file list.

Reply:
For this I made some changes plase review